docs/quick_start/quick_start_with_datagrip.md

```diff
@@ -23,7 +23,7 @@
 ### Get DataGrip And Install
 Please go to [Download DataGrip](https://www.jetbrains.com/datagrip/download) to get and install an appropriate version for yourself.
 ### Get Kyuubi Started
-[Get kyuubi server started](https://kyuubi.apache.org/docs/latest/quick_start/quick_start.html) before you try DataGrip with kyuubi.
+[Get kyuubi server started](../quick_start/quick_start.md) before you try DataGrip with kyuubi.
```

Comment:
`./quick_start.html` ?

Reply:
We currently do not have a uniform suffix, using `. Md` works without `#` behind.

Reply:
oh sorry, I mean `./quick_start.md`

Reply:
Now Kyuubi Doc is having a problem with 404 when it needs to specify a title to jump to if the suffix is `.md`.
So, do we need to unify the suffix, and if we do, it's just `.html`.